pandas/parser.pyx

```diff
@@ -1239,6 +1239,10 @@ class CParserError(ValueError):
     pass
 
 
+class EmptyDataError(ValueError):
+    pass
```

Comment:
yes that too (though odd why we aren't just using plan ``OverflowError``) which is a standard exception.